crypto/rand/rand_unix.c

```diff
@@ -61,32 +60,24 @@ int RAND_poll(void)
     int i, k;
     struct timespec ts;
     unsigned char v;
-
 #  ifdef OPENSSL_SYS_VOS_HPPA
     long duration;
     extern void s$sleep(long *_duration, short int *_code);
 #  else
-#   ifdef OPENSSL_SYS_VOS_IA32
     long long duration;
     extern void s$sleep2(long long *_duration, short int *_code);
-#   else
-#    error "Unsupported Platform."
-#   endif                       /* OPENSSL_SYS_VOS_IA32 */
-#  endif                        /* OPENSSL_SYS_VOS_HPPA */
+#  endif
 
     /*
      * Seed with the gid, pid, and uid, to ensure *some* variation between
      * different processes.
      */
-
     curr_gid = getgid();
     RAND_add(&curr_gid, sizeof curr_gid, 1);
```

Comment:
They serve to distinguish different instances of OpenSSL in the absence of other randomness.
They're reasonable to include at start up but not useful for later updates.

There are a pile of other things that can be used as well.  For example `gettid(2)` and `sysinfo(2)` but they vary a lot system to system.